frontends/mit-open/src/pages/FieldPage/FieldSearchFacetDisplay.tsx

```diff
@@ -96,22 +77,48 @@ const AvailableFacetsDropdowns: React.FC<
           constantSearchParams[facetSetting.name as FacetKey] || null,
         )
 
+        const isMultiple = BOOLEAN_FACET_NAMES.includes(facetSetting.name)
+          ? false
+          : true
+
+        let displayValue
+        if (BOOLEAN_FACET_NAMES.includes(facetSetting.name)) {
+          displayValue =
+            activeFacets[facetSetting.name as BooleanFacetKey] === true ||
+            activeFacets[facetSetting.name as BooleanFacetKey] === false
+              ? (
+                  activeFacets[facetSetting.name as BooleanFacetKey] as boolean
+                ).toString()
+              : ""
+        } else {
+          displayValue = activeFacets[facetSetting.name as FacetKey] || []
+        }
+
         return (
           facetItems.length > 0 && (
             <FormControl key={facetSetting.name}>
               <Select
-                multiple
+                multiple={isMultiple}
                 displayEmpty
-                value={activeFacets[facetSetting.name as FacetKey] || []}
+                value={displayValue}
                 renderValue={() => {
                   return facetSetting.title
                 }}
-                onChange={getHandleChangeForFacet(
-                  activeFacets[facetSetting.name as FacetKey] || [],
-                  facetSetting.name,
-                )}
+                onChange={(e) =>
+                  onFacetChange(facetSetting.name, e.target.value)
+                }
                 sx={{ m: 1, minWidth: 140 }}
               >
+                {!isMultiple ? (
+                  <MenuItem
+                    value=""
+                    key={facetSetting.name.concat(":", "unselect")}
+                  >
+                    no selection
```

Comment:
"All" might be a good name for "no filters".

Reply:
sure - I can change it to All

Reply:
decided not to change - we will update everything when there is a design